app/views/admin/members/_profile.html.haml

```diff
@@ -1,47 +1,47 @@
 #profile
-  = image_tag(@member.avatar(200), class: 'th', title: @member.full_name, alt: @member.full_name)
-  .name= @member.full_name
-  .email= mail_to @member.email, @member.email
-  %dt= @member&.mobile
-  %p.lead= @member.about_you
-
-  %hr
+  = image_tag(@member.avatar(200), title: @member.full_name, alt: @member.full_name)
+  .div= @member.full_name
```

Comment:
@KimberleyCook this will create a `div` with `class="div"` on it? Is this what you want? If you just want a div element this should be `%div`